website/content/_index.md

```diff
@@ -120,12 +123,89 @@ Config:
 
 We can interrogate the container more closely with `wexec`
 ```
-wash$ wexec 45a0265546d63a8f1b0d17033748db1468dc49dfd09cdaf2db62c45a60e82aaf whoami
+wash$ wexec swarm_web_1 whoami
 root
 ```
 
 Try exploring `docker/volumes` to interact with the volume created for Redis.
 
+### Finding with metadata
+
+Wash includes a powerful `find` command that can match based on an entry's metadata. For example, if we wanted to see what containers were created today, we would look at the `.Created` entry for Docker containers and the `.metadata.creationTimestamp` for Kubernetes pods. We can find all instances created in the last 24 hours with
+
+```
+find -meta .Created -24h -o -meta .metadata.creationTimestamp -24h
```

Comment:
Nit: This would need to set the `daystart` option to work properly. Maybe phrase the example as "instances created in the last 24 hours" instead of "today" ?

Reply:
I do: `We can find all instances created in the last 24 hours with`

The "created today" phrasing is accurate, but maybe throwing you off because I then use a slightly different example.

Reply:
Changed it to say "created recently".